Doc/library/idle.rst

```diff
@@ -689,7 +689,7 @@ When IDLE first starts, it attempts to read user configuration files in
 ``~/.idlerc/`` (~ is one's home directory).  If there is a problem, an error
 message should be displayed.  Leaving aside random disk glitches, this can
 be prevented by never editing the files by hand, using the configuration
```

Comment:
I think this change still leaves the grammar of the sentence mangled. It should be something like "by never editing the files by hand, instead using the configuration dialog under Options."

Reply:
Thank you for seeing what I was trying to say.  I made this change, with '.' instead of ','.